packages/backend/src/open_payments/payment/incoming/model.ts

```diff
@@ -197,14 +212,14 @@ export class IncomingPayment
             ...json.incomingAmount,
             value: json.incomingAmount.value.toString()
           }
-        : null,
+        : undefined,
```

Comment:
`undefined` is not proper JSON, but `null` is. I'd prefer keeping it that way. Same goes for lines 221 and 222.

Reply:
You've brought this up before, and I forgot.
I dropped the `null` to `undefined` commit.

I might keep exploring a comparable change though. My intent is to have representations of the schemas in our OpenAPI spec, and `null` is distinct from a property not being required.
https://json-schema.org/understanding-json-schema/reference/null.html
https://json-schema.org/understanding-json-schema/reference/object.html#required

Reply:
https://www.typescriptlang.org/tsconfig#exactOptionalPropertyTypes :eyes:

Reply:
- https://github.com/interledger/rafiki/issues/594